scripts/install.sh

```diff
@@ -5,27 +5,66 @@ CONFIG_DIR="/etc/crowdsec/cs-firewall-bouncer/"
 PID_DIR="/var/run/crowdsec/"
 SYSTEMD_PATH_FILE="/etc/systemd/system/cs-firewall-bouncer.service"
 
-check_iptables() {
+# Default pkg manager is apt
+PKG="apt"
+# Default firewall backend is nftables
+FW_BACKEND="nftables"
+API_KEY=""
+
+check_pkg_manager(){
+    if [ -f /etc/redhat-release ] ; then
+        PKG="yum"
+    fi   
```

Comment:
can we do the same thing with `/etc/debian_version` so we can tell the user beforehand "we don't know your distribution, things might go wrong ?"